File: app/src/main/java/com/user/canopas/dagger/mvp/view/ActorView.java
package com.user.canopas.dagger.mvp.view;


import com.user.canopas.dagger.mvp.model.ActorRespone;

import rx.Observable;

public interface ActorView {


    void compelet();

    void Error(String message);

    void GetActorResponse(ActorRespone actorRespone);



}

File: app/src/main/java/com/user/canopas/dagger/mvp/view/ActorView.kt
package com.user.canopas.dagger.mvp.view


import com.user.canopas.dagger.mvp.model.ActorRespone

import rx.Observable

interface ActorView {


    fun compelet()

    fun Error(message: String?)

    fun GetActorResponse(actorRespone: ActorRespone)


}
